mlflow_elasticsearchstore/models.py

```diff
@@ -128,3 +129,61 @@ def to_mlflow_entity(self) -> Run:
             params=[p.to_mlflow_entity() for p in self.params],
             tags=[t.to_mlflow_entity() for t in self.tags])
         return Run(run_info=run_info, run_data=run_data)
+
+
+class ExperimentIndex():
+    name = "mlflow-experiments"
+    settings = {
+        "number_of_shards": 1,
+        "number_of_replicas": 1
+    }
+    mappings = {
+        "properties": {
+            "name": {"type": "keyword"},
+            "artifact_location": {"type": "text"},
+            "lifecycle_stage": {"type": "keyword"},
+            "tags": {"type": "flattened"}
+        }
+    }
+
+    @classmethod
+    def init(cls, indices: IndicesClient) -> None:
+        if not (indices.exists(index=[cls.name])):
+            indices.create(index=cls.name,
+                           body={"settings": cls.settings,
+                                 "mappings": cls.mappings})
+
+
+class RunIndex():
+    name = "mlflow-runs"
+    settings = {
+        "number_of_shards": 2,
```

Comment:
Why do we chose these values for shards and replicas ? (and why 1 only in the ExperimentIndex ?)

Reply:
Run Index is bigger than Experiment Index but values are a bit random

Reply:
This explanation fits for the number of shards, but why do you replicate the runs and not the experiments ?